ajax/libs/vega-lite/package.json

```diff
@@ -16,6 +16,12 @@
       "files": [
         "vega-lite*.+(js|css|map)"
       ]
+    },
+    {
+      "basePath": "build",
+      "files": [
+        "**/vega-lite*.+(js|css|map)"
```

Comment:
@domoritz I'd like to make sure If there are only distributed purpose files under `build` directory. If so, we can use `"**/*"` instead. If not, let's just keep this.